index.html

```diff
@@ -45,7 +45,7 @@
       The GNU GPL (<a href="licenses/gpl-2.0/">V2</a> or <a href="licenses/gpl-3.0/">V3</a>) is a copyleft license that requires anyone who distributes your code or a derivative work to make the source available under the same terms. V3 is similar to V2, but further restricts use in hardware that forbids software alterations.
     </p>
     <p>
-      <strong>WordPress</strong>, <strong>Linux</strong>, and <strong>Bash</strong> use the GNU GPL.
+      <a href="https://github.com/WordPress/WordPress/blob/master/license.txt">WordPress</a>, <a href="https://github.com/torvalds/linux/blob/master/COPYING">Linux</a>, and <a href="https://www.gnu.org/software/bash/">Bash</a> use the GNU GPL.
```

Comment:
I don't see any mentions of “mirror” in the GitHub page, but I'd guess a better canonical link for Linux would be [this][1].

[This][2] looks like a good link for Bash.  [Chet doesn't actually develop in public][3], but that repo is where things get thrown over the wall.

[1]: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/COPYING
[2]: http://git.savannah.gnu.org/cgit/bash.git/tree/COPYING
[3]: http://thread.gmane.org/gmane.comp.shells.bash.bugs/13695/focus=13827